stdlib/2and3/math.pyi

```diff
@@ -29,7 +29,7 @@ def exp(x: float) -> float: ...
 def expm1(x: float) -> float: ...
 def fabs(x: float) -> float: ...
 def factorial(x: int) -> int: ...
-def floor(x: float) -> float: ...
+def floor(x: float) -> int: ...
```

Comment:
:| weird, messed up then. Should I just copy this file to 3/ and change it there?